front/src/cable.ts

```diff
@@ -0,0 +1,6 @@
+// react-frontend/src/cable.ts
+import { createConsumer } from '@rails/actioncable';
+const URL = 'ws://localhost:3000/cable'; // What to replace localhost:3000 with on production?
```

Comment:
This seems to be doing 'something', as if I change the URL to anything else (e.g. `ws://localhost:4000/cable`), the Rails logs spam a warning...
```Ruby
cl-back-web    | 2023-04-06 13:31:23.587642 E [10:worker-1 tagged_logger_proxy.rb:38] ActionCable -- Subscription class not found: "users_blocked_count_channel"
```